data/data/openstack/variables-openstack.tf

```diff
@@ -10,12 +10,16 @@ variable "openstack_master_root_volume_size" {
   description = "The size of the volume in gigabytes for the root block device of master nodes."
 }
 
-variable "openstack_base_image" {
+variable "openstack_base_image_name" {
   type        = string
-  default     = "rhcos"
   description = "Name of the base image to use for the nodes."
 }
 
+variable "openstack_base_image_url" {
+  type        = string
```

Comment:
We need a default value, otherwise it fails with `The input variable "openstack_base_image_url" has not been assigned a value.` when exporting `OPENSHIFT_INSTALL_OS_IMAGE_OVERRIDE`.

Reply:
Funny thing, I did that but forgot to `git commit --amend` it and pushed the patch without this change. Agree, this is important.